### PR TITLE
Updated footer in vue & ruby

### PR DESCRIPTION
**Pull request by rsgonzal**
Updating footers in location where i've found them

### CODE REVIEW

**Comment by rsgonzal** [comment]
I may have been way off with there 😅

File: app/views/application_rebrand/_footer.en.html.erb [hunk 1]
```diff
@@ -9,19 +9,39 @@
...
-        <a href="#">Contact</a>
-        <a href="#">Terms of Use</a>
-        <a href="#">Press</a>
-        <a href="#">Privacy Policy</a>
-        <a href="#">Donate</a>
+        <%= link_to "FAQ",
+            "https://iridescentsupport.zendesk.com/hc/en-us/categories/115000091348-Technovation"
+          %>
+        <%= link_to "Contact",
+            "mailto:support@technovation.org"
```
Comment (shaunxp20) [inline app/views/application_rebrand/_footer.en.html.erb:31]: I think this is in a config/ENV setting, anywho, if so, no worries, we can update it later! 😃

**Review by shaunxp20 — APPROVED** [review]
Amazing! Thanks for updating all of this! Looks good to me! Feel free to merge, I updated it, so it's up to date w/ QA. :shipit: 👍 😀

File: app/views/application_rebrand/_footer.en.html.erb (hunk 1 above)
Comment (rsgonzal) [reply app/views/application_rebrand/_footer.en.html.erb:31]: it is! but i wasn't sure how to do that! i was also trying to make the year in the footer pull from config, but just couldn't figure it out.. 😅

**Pull request merged**